incubator/postfix/templates/secrets.yaml

```diff
@@ -9,6 +9,11 @@ metadata:
     heritage: "{{ .Release.Service }}"
 type: Opaque
 data:
-  smtp_username: {{ trim .Values.smtp.username | b64enc }}
   smtp_password : {{ trim .Values.smtp.password | b64enc }}
   smtp_domain: {{ trim .Values.smtp.domain | b64enc }}
```

Comment:
Need to revert this back
RELAY_AUTH_ENABLED 
This is required by postfix docker image.